scripts/listen_to_events/strategies.py

```diff
@@ -203,6 +224,9 @@ class AmpUpdateStoppedStrategy(EventStrategy):
     async def format_topics(self, chain, event):
         return {k: v for k, v in parse_event_topics(event).items()}
 
+    async def get_pool_address(self, event):
+        return event["address"]
```

Comment:
don't use deceitful function names
```suggestion
    async def get_event_address(self, event):
        return event["address"]
```

Reply:
but not always the pool and the event will be the same address. For example, the factory emits an event of a new pool being created.

Reply:
right, that's a bit my point 👀 this function doesn't know what it's extracting from the event.